lib/feature/home_page/home_page.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:counter_async_redux/api/models/pokemon_model.dart';
+import 'package:counter_async_redux/feature/home_page/widgets/pokemon_tile.dart';
+
+import 'package:flutter/material.dart';
+
+class HomePage extends StatelessWidget {
+  const HomePage({
+    required this.getPokemon,
+    required this.pokemon,
+    this.next,
+  });
+
+  final Function(bool? isScrolling) getPokemon;
+  final List<Pokemon> pokemon;
+  final String? next;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
why do you have two scaffolds and two app bars?

Reply:
@Enrickzz

Reply:
Separate header and search bar. Will find a better way.